### PR TITLE
OneScript 1.8.0

### PR DESCRIPTION
**Pull request by dmpas**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/70000)

### CODE REVIEW

**Comment by dmpas** [comment]
https://github.com/EvilBeaver/OneScript  - here we have a link to oscript.io site
https://oscript.io/dev/sources - here we have a link to the github page
`oscript` is a shortcut for `OneScript`

does this suit as an evidence ?

File: manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.8.0
+PackageLocale: en-US
+Publisher: 1Script Team (Open Source)
+PackageName: OneScript
```
Comment (ItzLevvie) [inline manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.en-US.yaml:7]:
```suggestion
PackageName: OneScript engine
```
This must match Control Panel.

File: manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.ru-RU.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.8.0
+PackageLocale: ru-RU
+Publisher: 1Script Team (Open Source)
+PackageName: OneScript
```
Comment (ItzLevvie) [inline manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.ru-RU.yaml:7]:
```suggestion
PackageName: OneScript engine
```
This must match Control Panel.

File: manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.8.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.0.0
```
Comment (ItzLevvie) [inline manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.yaml:7]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.ru-RU.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+License: MPL-2.0
+ShortDescription: Движок OneScript
+Tags:
+- "console"
+- "command-line"
+- "onescript"
+- "oscript"
+- "оскрипт"
+ManifestType: locale
+ManifestVersion: 1.0.0
```
Comment (ItzLevvie) [inline manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.ru-RU.yaml:17]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+License: MPL-2.0
+ShortDescription: OneScript engine
+Tags:
+- "console"
+- "command-line"
+- "onescript"
+- "oscript"
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0
```
Comment (ItzLevvie) [inline manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.en-US.yaml:16]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  InstallerUrl: https://oscript.io/downloads/1_8_0/x64/OneScript-1.8.0-x64.exe
+  InstallerSha256: 307232F016F8C82A01AC47E989B06A3A9EC6C36122F8A12EC9087FE33AF5550A
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://oscript.io/downloads/1_8_0/OneScript-1.8.0-x86.exe
+  InstallerSha256: 6F8DAA79C83B575628A8B18A50984470837EA2003DCFCB4C92E997EB81FF0C96
+InstallerSwitches:
+  Silent: /verysilent /norestart
+ManifestType: installer
+ManifestVersion: 1.0.0
```
Comment (ItzLevvie) [inline manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.installer.yaml:20]:
```suggestion
ManifestVersion: 1.1.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://oscript.io/downloads/1_8_0/x64/OneScript-1.8.0-x64.exe
+  InstallerSha256: 307232F016F8C82A01AC47E989B06A3A9EC6C36122F8A12EC9087FE33AF5550A
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://oscript.io/downloads/1_8_0/OneScript-1.8.0-x86.exe
+  InstallerSha256: 6F8DAA79C83B575628A8B18A50984470837EA2003DCFCB4C92E997EB81FF0C96
+InstallerSwitches:
+  Silent: /verysilent /norestart
```
Comment (ItzLevvie) [inline manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.installer.yaml:18]:
```suggestion
  Custom: /NORESTART
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**